### DISCUSSION TITLE
More papers that are not yet included in the bibliography

### DISCUSSION THREAD

**Discussion by amirouche** [schemedoc/bibliography#9] [Ideas]
ORBIT: An Optimizing Compiler For Scheme by David Andrew Kranz
https://cpsc.yale.edu/sites/default/files/files/tr632.pdf

**Comment by lassik** [comment]
Implicit Phasing for R6RS Libraries
https://guenchi.github.io/Scheme/doc/Implicit%20Phasing%20for%20R6RS%20Libraries.pdf

**Comment by lassik** [comment]
Continuation-passing, closure-passing style (1989)
https://users.soe.ucsc.edu/~lkuper/papers/lambdae.pdf

**Comment by lassik** [comment]
An Adaptive Package Management System for Scheme
https://www.stklos.net/~eg/Publis/dls07.pdf

**Comment by lassik** [comment]
Reverse-Mode AD in a Functional Framework: Lambda the Ultimate Backpropagator
http://www.bcl.hamilton.ie/~barak/papers/toplas-reverse.pdf

**Comment by omasanori** [comment]
Moved to discussions as this topic is not a kind of "OK, we've done so let's close it" thing.
